src/main/java/CommandDelete.java

```diff
@@ -0,0 +1,20 @@
+public class CommandDelete extends Command {
```

Comment:
Shouldn't there be Javadoc comments on the CommandDelete class and its public methods? ...I've also noticed a few other classes which are missing Javadoc comments.